docs/articles/documentation/recipes/using-page-model.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: docs
+title: Using Page Model
+permalink: /documentation/recipes/using-page-model.html
+---
+# Using Page Model
+
+Page Model is a test automation pattern that allows you to create an abstraction of the tested page
+and use it in test code to refer to page elements.
+
+## Why Use Page Model
+
+Consider the following fixture with a test that types and edits the developer name on the [example](https://devexpress.github.io/testcafe/example/) webpage.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `My fixture`
+    .page `https://devexpress.github.io/testcafe/example/`;
+
+const nameInput = Selector('#developer-name');
+
+test('Text typing basics', async t => {
```

Comment:
I guess it's better to start with an example where you use inline selectors, e.g. `t.click('#developer-name')`. One of the biggest advantages of PageModel is the abstraction from the actual page markup. With inline selectors you'll need to change selectors in each test, with PageModel you will need to modify selector only in single place.